dcos/marathon.py

```diff
@@ -669,6 +669,37 @@ def get_task(self, task_id):
 
         return task
 
+    def stop_task(self, task_id, wipe=None):
+        """Stops a task.
+
+        :param task_id: the ID of the task
+        :type task_id: str
+        :param wipe: whether remove reservations and persistent volumes.
+        :type wipe: bool
+        :returns: a tasks
+        :rtype: dict
+        """
+
+        if not wipe:
+            params = None
+        else:
+            params = {'wipe': 'true'}
```

Comment:
I don't think so. the default behavior of `delete` / `stop` simply kills the task. Adding `--wipe` already is the confirmation that the task should be wiped off the state. (We also don't ask to confirm a `dcos marathon app remove`, in which case the task will also be wiped.)